projects/ion/src/lib/smart-table/smart-table.component.ts

```diff
@@ -35,6 +35,7 @@ export class IonSmartTableComponent<RowType>
 
   public mainCheckBoxState: CheckBoxStates = 'enabled';
   public pagination!: PageEvent;
+  public openItemsPerPageAbove = false;
```

Comment:
Perhaps you could avoid this aditional propery and set the property from the config directly on the html file with all the validations.
```suggestion
```